src/tests/automated_tests/conftest.py

```diff
@@ -1,3 +1,9 @@
+#
+#  Copyright (c) 2019-2020, ETH Zurich. All rights reserved.
```

Comment:
```suggestion
#  Copyright (c) 2019-2021, ETH Zurich. All rights reserved.
```

Reply:
And same for the rest of the automated tests.

Reply:
Actually, those files were written in 2020. The ones I should change to 2021 are in `deploy/test-build/cluster`.

Reply:
Looking at the documentation on copyright, all sources should be updated until 2021. Done in https://github.com/eth-cscs/firecrest/pull/50/commits/7abdb22e9e996735917473c669e5c10f3d4ea280